specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2015-01-01/resourcehealth.json

```diff
@@ -221,6 +222,154 @@
         }
       }
     },
+    "/{resourceUri}/providers/Microsoft.ResourceHealth/child/availabilityStatuses/current": {
```

Comment:
The "child" token in these paths is out of spec with ARM contract. If this a singleton - and there will not be many child instances-, this is more appropriate:
`/{resourceUri}/providers/Microsoft.ResourceHealth/childAvailabilityStatuses/current`